qcodes/instrument_drivers/HP/HP8753D.py

```diff
@@ -1,331 +1,5 @@
-
-        # set a longer timeout, to not timeout during the sweep
-        new_timeout = st*N + 2
-
-        with self.timeout.set_to(new_timeout):
-            log.debug(f'Making {N} blocking sweeps.' +
-                      f' Setting VISA timeout to {new_timeout} s.')
-
-            self.ask(f'OPC?;NUMG{N}')
-
-    def invalidate_trace(self, cmd: str,
-                         value: Union[float, int, str]) -> None:
-        """
-        Wrapper for set_cmds that make the trace not ready
-        """
-        self._traceready = False
-        self.write(cmd.format(value))
-
-    def startup(self) -> None:
-        self._traceready = False
-        self.display_format(self.display_format())
-
-    def _s_parameter_setter(self, param: str) -> None:
-        """
-        set_cmd for the s_parameter parameter
-        """
-        if param not in ['S11', 'S12', 'S21', 'S22']:
-            raise ValueError('Cannot set s-parameter to {}')
-
-        # the trace labels changes
-        self._traceready = False
-
-        self.write(param)
-
-    def _s_parameter_getter(self) -> str:
-        """
-        get_cmd for the s_parameter parameter
-        """
-        for cmd in ['S11?', 'S12?', 'S21?', 'S22?']:
-            resp = self.ask(cmd)
-            if resp in ['1', '1\n']:
-                break
-
-        return cmd.replace('?', '')
-
-    def _display_format_setter(self, fmt: str) -> None:
-        """
-        set_cmd for the display_format parameter
-        """
-        val_mapping = {'Log mag': 'LOGM',
-                       'Phase': 'PHAS',
-                       'Delay': 'DELA',
-                       'Smith chart': 'SMIC',
-                       'Polar': 'POLA',
-                       'Lin mag': 'LINM',
-                       'Real': 'REAL',
-                       'Imaginary': 'IMAG',
-                       'SWR': 'SWR'}
-
-        if fmt not in val_mapping.keys():
-            raise ValueError(f'Cannot set display_format to {fmt}.')
-
-        self._traceready = False
-        self.display_reference.unit = _unit_map[fmt]
-        self.display_scale.unit = _unit_map[fmt]
-
-        self.write(val_mapping[fmt])
-
-    def _display_format_getter(self) -> str:
-        """
-        get_cmd for the display_format parameter
-        """
-        val_mapping = {'LOGM': 'Log mag',
-                       'PHAS': 'Phase',
-                       'DELA': 'Delay',
-                       'SMIC': 'Smith chart',
-                       'POLA': 'Polar',
-                       'LINM': 'Lin mag',
-                       'REAL': 'Real',
-                       'IMAG': 'Imaginary',
-                       'SWR': 'SWR'}
-
-        # keep asking until we find the currently used format
-        for cmd in val_mapping.keys():
-            resp = self.ask(f'{cmd}?')
-            if resp in ['1', '1\n']:
-                break
-
-        return val_mapping[cmd]
+from HP_8753D import HP8753D
```

Comment:
we probably need to import these as well, just in case, for backwards compatibility
```suggestion
from HP_8753D import HP8753D, HP8753DTrace, TraceNotReady
```